ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -149,9 +162,50 @@ nodeToNodeCodecCBORTerm version
                              InitiatorOnlyDiffusionMode         -> True
                              InitiatorAndResponderDiffusionMode -> False)
               , CBOR.TInt (case peerSharing of
-                             NoPeerSharing      -> 0
-                             PeerSharingPrivate -> 1
-                             PeerSharingPublic  -> 2)
+                             PeerSharingDisabled -> 0
+                             PeerSharingEnabled  -> 1)
+              , CBOR.TBool query
+              ]
+
+        decodeTerm :: NodeToNodeVersion -> CBOR.Term -> Either Text NodeToNodeVersionData
+        decodeTerm _ (CBOR.TList [CBOR.TInt x, CBOR.TBool diffusionMode, CBOR.TInt peerSharing, CBOR.TBool query])
+          | x >= 0
+          , x <= 0xffffffff
+          , peerSharing >= 0
+          , peerSharing <= 2
```

Comment:
It should be smaller or equal than `1`, isn't it?
```suggestion
          , peerSharing <= 1
```

Reply:
Yes :)